distrobox-init

```diff
@@ -169,7 +169,7 @@ while :; do
 done
 
 # Check we're running inside a container and not on the host
-if [ ! -f /run/.containerenv ] && [ ! -f /.dockerenv ]; then
+if [ ! -f /run/.containerenv ] && [ ! -f /.dockerenv ] && [ -z "${container}" ] && [ -z "${CONTAINER_ID:-}" ]; then
```

Comment:
Here we can just check $container  (and the files for backward compatibility)
As we inject `container` at creation (see above) 

CONTAINER_ID will not be present here in the init as it's injected at `enter` time